src/LoggedIn.js

```diff
@@ -31,20 +51,88 @@ class LoggedIn extends Component {
       });
   }
 
+  createNote(event) {
+    const id = uuid();
+    firebase.database().ref(`notes/${id}`).set(this.generateNote());
+    this.setState({ currentNote: id });
+  }
+
+  generateNote() {
+    const title = `The ${this.generateRandomWord(adjectives)} ${this.generateRandomWord(animals)}`;
+    const body = `${title} ${this.generateRandomWord(verbs)} the ${this.generateRandomWord(nouns)} and exclaimed "${this.generateRandomWord(exclamations)}"!`;
+    return { title, body };
+  }
+
+  generateRandomWord(words) {
+    return words[Math.floor(Math.random() * Math.floor(words.length))];
+  }
+
+  updateNote(event, noteId) {
+    event.preventDefault();
+    firebase.database().ref(`notes/${noteId}`).update(this.generateNote());
+  }
+
+  deleteNote(event, noteId) {
+    event.preventDefault();
+    firebase.database().ref(`notes/${noteId}`).remove();
+    const notes = this.state.notes;
+    delete notes[noteId];
+    const first = Object.keys(notes)[0];
+    this.setState({ notes, currentNote: first });
+  }
+
+  selectNote(event, noteId) {
+    event.preventDefault();
+    this.setState({ currentNote: noteId });
+  }
+
+  renderList() {
+    const { notes } = this.state;
+    const selectNote = this.selectNote;
+    const currentNoteId = this.state.currentNote;
+    return (
+      <ul className="list-group list-group-flush">
+        {Object.keys(notes).map((noteId, index) => {
+          return (
+            <div key={index}>
+              <button type="button" className={currentNoteId === noteId ? 'list-group-item list-group-item-action active' : 'list-group-item list-group-item-action'} onClick={(event) => selectNote(event, noteId)}>{notes[noteId].title}</button>
+            </div>
+          )
+        })}
+      </ul>
+    );
+  };
+
   render() {
-    const user = this.currentUser;
-    const userDetails = Object.keys(this.currentUser).map(function(key) {
-      return <div><p>{key}: {user[key]}</p></div>
-    });
+    const currentNoteId = this.state.currentNote;
+    const currentNote = this.state.notes[currentNoteId];
 
     return (
-      <div className="container">
-      <div className="jumbotron">
-        <div className="text-center">
-          <p>Welcome {this.currentUser.email}!</p>
+      <div className="row">
```

Comment:
You could consider making a `max-width` on this outermost element to keep the view uniform no matter how wide the view is. Just a 🤔